src/Models/Log.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Bonnier\WP\Redirect\Models;
+
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Support\Arr;
+
+class Log implements Arrayable
+{
+    /** @var int */
+    private $logID;
+    /** @var string|null */
+    private $slug;
+    /** @var string|null */
+    private $hash;
+    /** @var string|null */
+    private $type;
+    /** @var int */
+    private $wpID;
+    /** @var \DateTime */
+    private $createdAt;
+
+    public function __construct()
+    {
+        $this->logID = 0;
+        $this->wpID = 0;
+        $this->createdAt = new \DateTime();
+    }
+
+    /**
+     * @return int
+     */
+    public function getID(): int
+    {
+        return $this->logID;
+    }
+
+    /**
+     * @param int $logID
+     * @return Log
+     */
+    public function setID(int $logID): Log
+    {
+        $this->logID = $logID;
+        return $this;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getSlug(): ?string
+    {
+        return $this->slug;
+    }
+
+    /**
+     * @param string|null $slug
+     * @return Log
+     */
+    public function setSlug(?string $slug): Log
```

Comment:
Perhaps make the setter use `UrlHelper::normalizePath()` so you dont have to call it outside?